src/settings/VideoSettings.tsx

```diff
@@ -18,6 +18,10 @@ const obsResolutions: any = ipc.sendSync('settingsWindow', ['getObsAvailableReso
 const { Base: baseResolutions, Output: outputResolutions } = obsResolutions;
 
 const fpsOptions = ['10', '20', '30', '60'];
+const obsCaptureModes = {
+  'game_capture': 'Game (best)',
```

Comment:
Can we reword this user facing text to:

`Game Capture (Recommended)`
`Monitor Capture`

Reply:
We can do whatever you want! =)